docs/developer-docs/backend/motoko/index.md

```diff
@@ -1,10 +1,16 @@
+# Introduction to Developing Canisters in Motoko
+
+[Motoko](/motoko/main/motoko.md) was [specifically designed](https://stackoverflow.blog/2020/08/24/motoko-the-language-that-turns-the-web-into-a-computer/) by DFINITY to support the unique features of the Internet Computer and to provide a familiar yet robust programming environment.
+
+To get started, one should [install the `dfx` CDK](../../setup/install/index.mdx) which supports building Motoko [canister smart contracts](https://internetcomputer.org/how-it-works/architecture-of-the-internet-computer/#canister-smart-contracts).
```

Comment:
Still refers to dfx as a CDK

Reply:
good catch!

Reply:
so much surface area!

Reply:
(ironic since it was the ORIGINAL comment you had and i edited dozens of other files)

Reply:
done